src/components/common/Badge.tsx

```
@@ -0,0 +1,110 @@
+      return (
+        <span className={scheduleBadgeStyles({schedule: props.schedule})}>
+          {props.children}일 전
+        </span>
+      );
+
+    // 인덱스 배지 (단원, 문제 등)
+    case 'index':
+      const suffix = props.kind === 'unit' ? '단원' : ' 문제';
+
+      return (
+        <span className={indexBadgeStyles({kind: props.kind})}>
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)